src/ClockPanel.tsx

```diff
@@ -270,62 +232,50 @@ class UnthemedClockPanel extends PureComponent<Props, State> {
     `;
 
     const display = now.locale(dateSettings.locale || '').format(dateSettings.dateFormat);
-    
+
     return (
       <span>
         <h3 className={className}>{display}</h3>
       </span>
     );
   }
 
-  renderTime() {
-    const { now } = this.state;
-    const { options } = this.props;
-    const { timeSettings, mode } = options;
+  function renderTime() {
+    const { timeSettings, clockType, mode } = props.options;
 
     const className = css`
       font-size: ${timeSettings.fontSize};
       font-weight: ${timeSettings.fontWeight};
       margin: 0;
     `;
```

Comment:
Same thing here as mentioned above. Maybe use the `getStyles` convention instead of defining this inline.